apps/pie-docs/src/components/icon-button/code/code.md

```diff
@@ -0,0 +1,153 @@
+---
+eleventyNavigation:
+    key: Code
+    parent: Icon Button
+    order: 3
+shouldShowContents: true
+eleventyComputed:
+    props: "{% include './props.json' %}"
+---
+
+## Overview
+
+<p>
+  <a href="https://www.npmjs.com/@justeattakeaway/pie-icon-button">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-icon-button.svg">
+  </a>
+</p>
+
+`pie-icon-button` is a Web Component built using the Lit library. It offers a simple and accessible icon button component for web applications.
+
+This component can be easily integrated into various frontend frameworks and customized through a set of properties.
+
+## Installation
+
+To install `pie-icon-button` in your application, run the following on your command line:
+
+```shell
+# npm
+$ npm i @justeattakeaway/pie-icon-button
+```
+
+```shell
+# yarn
+$ yarn add @justeattakeaway/pie-icon-button
+```
+
+{% notification {
+  type: "neutral",
+  iconName: "link",
+  message: "For more information on using PIE components as part of an application, check out the [Getting Started Guide.](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components)."
+} %}
+
+## Playground
+
+ <iframe
+  src="https://webc.pie.design/?path=/story/icon-button--primary&viewMode=story&shortcuts=true&singleStory=true"
+  width="100%"
+  height="600px"
+  style="border: none; margin-top: 32px;"
+></iframe>
+
+## Variants
+
+ <iframe
+  src="https://webc.pie.design/?path=/docs/icon-button--variants&viewMode=story&shortcuts=true&singleStory=true"
+  width="100%"
+  height="600px"
+  style="border: none; margin-top: 32px;"
+></iframe>
+
+## Importing Icons
+
+We recommend using the `pie-icon-button` component alongside the `pie-icons-webc` package. This package contains all of the PIE iconset as importable web components and are built to work alongside our other PIE components.
```

Comment:
"All of" implies plural, "iconset" is singular. I would say either "the entire iconset" or "all of the icons".

Reply:
updated to use all of the PIE icons. 🤝